core/common/src/test/java/alluxio/exception/AlluxioExceptionTest.java

```diff
@@ -19,9 +19,15 @@
 public class AlluxioExceptionTest {
 
   @Test
-  public void unexpectedException() {
-    AlluxioTException tException = new AlluxioTException("testType", "testMessage", "testClass");
-    AlluxioException exception = AlluxioException.fromThrift(tException);
-    Assert.assertTrue(exception instanceof UnexpectedAlluxioException);
+  public void unknownException() {
+    try {
+      final String testMessage = "testMessage";
+      AlluxioTException tException = new AlluxioTException("testType", testMessage, "testClass");
+      AlluxioException exception = AlluxioException.fromThrift(tException);
+      Assert.assertEquals(testMessage, exception.getMessage());
+    } catch (Exception e) {
```

Comment:
to make sure we do not regress to the old behavior, which would call `RuntimeException`